src/app/(main)/dashboard/[teamId]/page.tsx

```diff
@@ -0,0 +1,20 @@
+import VoyageDashboardPage from "@/app/(main)/dashboard/components/voyage-dashboard/VoyageDashboardPage";
+import PreVoyageDashboard from "@/app/(main)/dashboard/components/pre-voyage-dashboard/PreVoyageDashboard";
+
+interface VoyageMemberDashboardPageProps {
+  params: {
+    teamId: string;
+  };
+}
+function VoyageMemberDashboardPage({ params }: VoyageMemberDashboardPageProps) {
+  // TODO: Mocked temporary value
```

Comment:
Need to add a check here to see if the user is part of an active voyage because currently, user can just type in some random team id and it'll redirect them to the voyage dashboard, which is not intended. You can make use of the getcurrentvoyageteam util function

somethng like this

```
const [user, error] = await getUser();
  const teamId = Number(params.teamId);
  const { currentTeam } = getCurrentVoyageTeam({ user, error, teamId });

  if (!currentTeam) {
    redirect(routePaths.dashboardPage());
  }
```